cmd/vulcan-prowler/main.go

```diff
@@ -44,26 +45,47 @@ var (
 	CISCompliance = report.Vulnerability{
 		Summary: "Compliance With CIS AWS Foundations Benchmark (BETA)",
 		Description: `<p>
+			The check did not received the security classification level
```

Comment:
`s/received/receive/`

Reply:
Maybe we could also drop the "level" word like in the other descriptions.

Reply:
Yes thanks